pkg/bootstrap/stage.go

```diff
@@ -166,17 +166,22 @@ func extractFromDir(dir, prefix string, img v1.Image, imgName string) error {
 	if err := extract(imgName, tempDir, prefix, r); err != nil {
 		return err
 	}
-	if err := os.Rename(tempDir, dir); err != nil {
+	if err := os.Rename(tempDir, dir); err == os.ErrExist {
+		logrus.Print(err)
```

Comment:
I don't think we should print a message if the directory already exists. It's going to exist most of the time.